scripts/h5toobj.py

```diff
@@ -186,21 +131,11 @@ def generate_chunks(shape, dtype, target_object_size, dims_to_split=(0, 1)):
     return tuple(chunks)
 
 
-def dask_from_chunks(chunks, out_name, base_name):
-    single_chunk = all(len(c) == 1 for c in chunks)
-    last_index_per_dim = [np.r_[0, np.cumsum(c)[:-1]][-1] for c in chunks]
-    widths = [len(str(i)) for i in last_index_per_dim]
+def dsk_from_chunks(chunks, out_name):
     keys = list(product([out_name], *[range(len(bds)) for bds in chunks]))
     all_slices = da.core.slices_from_chunks(chunks)
     for key, slices in zip(keys, all_slices):
-        index = [s.start for s in slices]
-        if single_chunk:
-            obj_name = base_name
-        else:
-            index_str = '_'.join(["{:0{width}d}".format(i, width=w)
-                                  for (i, w) in zip(index, widths)])
-            obj_name = base_name + '/' + index_str
-        yield key, slices, obj_name
+        yield key, slices
```

Comment:
How about just `return zip(keys, all_slices)` instead of looping over and yielding each element?

Reply:
Yes, that was my thinking too.

Reply:
Done.